query-algebrizer/src/types.rs

```diff
@@ -691,6 +693,22 @@ impl Debug for EmptyBecause {
     }
 }
 
+
+/// A `FindQuery` represents a valid query to the query algebrizer.
+#[allow(dead_code)]
+#[derive(Debug, Eq, PartialEq)]
+pub struct FindQuery {
```

Comment:
It'd be nice to have a duplicate comment here, similar to one above ParsedFindQuery

Reply:
Sure.